openff/toolkit/tests/test_molecule.py

```diff
@@ -890,7 +890,6 @@ def compare_mols(ref_mol, inchi_mol):
 
     # TODO: Should there be an equivalent toolkit test and leave this as an integration test?
     @requires_openeye
-    @pytest.mark.slow
```

Comment:
This test isn't actually very slow and it covers some basic `Molecule` creation, so I think it's worth running more often than in nightly tests.

```
$ pytest openff/toolkit/tests/test_molecule.py::TestMolecule::test_create_from_file --runslow --durations=0 | tail -n1
======================== 1 passed, 3 warnings in 6.41s =========================